autogen/main/cluster.tf.tmpl

```diff
@@ -68,7 +68,8 @@ resource "google_container_cluster" "primary" {
 
 {% if beta_cluster %}
   cluster_autoscaling {
-    enabled = var.cluster_autoscaling.enabled
+    enabled             = var.cluster_autoscaling.enabled
+    autoscaling_profile = var.cluster_autoscaling.autoscaling_profile
```

Comment:
Should we set a default here (BALANCED)?

Reply:
I have the default in the cluster_autoscaling to BALANCED.

Reply:
Right, but that default will only be applied if `cluster_autoscaling` is omitted entirely. I should be able to specify `autoscaling_profile = null` to get a sane default (or should the default be disabled?).

Reply:
ah, yes good point; will update :)